### PR TITLE
fix string concatenation

### PR DESCRIPTION
**Pull request by parthshyara**
- Fix: strings concatenation for frozen string instead of `<<`
- Chore: add test for `fix` command in CI

### CODE REVIEW

**Comment by parthshyara** [comment]
address #92

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -43,5 +44,6 @@ jobs:
       - run: ls -lah ./pkg/apt-spy2*
       - run: gem install pkg/apt-spy2*
       - run: apt-spy2 check --strict
+      - run: sudo env "PATH=$PATH" apt-spy2 fix --commit --strict
```
Comment (till) [inline .github/workflows/pr.yml:47]:
```suggestion
      - run: sudo env "PATH=$PATH" apt-spy2 fix --commit --strict
      - run: apt-get update
```

**Review by till — COMMENTED** [review]
Can you add this as well? I should have added another test after running rubocop there.

File: .github/workflows/pr.yml (hunk 1 above)
Comment (parthshyara) [reply .github/workflows/pr.yml:47]: @till What are we testing with adding this step here?

Comment (till) [reply .github/workflows/pr.yml:47]: If update still works as expected. This would have caught the frozen string stuff. Not ideal, but it works?

Comment (parthshyara) [reply .github/workflows/pr.yml:47]: The `commit` command would've caught the frozen string error, while the update wouldn't have! Although, with `update`, we'd know that mirror picked during CI works, which will be a more thorough test but is a non-deterministic  one, so not really sure if it adds any value. What do you think?

**Pull request merged**